handler/src/main/java/io/netty/handler/ssl/OpenSslClientContext.java

```diff
@@ -165,7 +166,7 @@ public OpenSslClientContext(File certChainFile, TrustManagerFactory trustManager
      * @deprecated use {@link SslContextBuilder}
      */
     @Deprecated
-    public OpenSslClientContext(File trustCertChainFile, TrustManagerFactory trustManagerFactory,
+    public OpenSslClientContext(File trustCertCollectionFile, TrustManagerFactory trustManagerFactory,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Constructor has 11 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS00107)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "OpenSslClientContext" is 35 which is greater than 25 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)